docs/api-readme.md

```diff
@@ -0,0 +1,35 @@
+# LitElement API Documentation
+
+## Install @polymer/lit-element
+
+```
+npm install --save @polymer/lit-element
```

Comment:
This is where I'd like to include the version once we release 1.0: `npm install --save @polymer/lit-element@^1.0.0`